### DISCUSSION TITLE
[Low Priority] Ingredient Auto-complete / Catalogue of ingredients

### DISCUSSION THREAD

**Discussion by Pheggas** [mealie-recipes/mealie#782] [Feature Request]
First option:
 I'd like to have database of all ingredients for auto-completing word. For example, in Ingredients section, i want to add "flour". So i start typing "fl" and with the gray-ish color it should predict what you want to write. 
Then, with the TAB, you'll approve system's prediction and it will complete the word without needing to manually complete it.

Second option:
We'll have list of all ingredients and via drag-n-drop function, we'll be able to add new ingredient, depending on which we have chosen.

It is really like final feature so i think the priority isn't as high but it is still something i'd like to see in Mealie. Especially older people would appreciate this feature i think.

**Comment by hay-kot** [comment]
Can you review the beta site and see if that fulfills this request? I'm not sure i 100% understand, but I think that the new implementation of ingredients does cover this. 

https://beta.mealie.io/login

**Reply by Pheggas** [reply]
Do you mean the Parser?

**Reply by hay-kot** [reply]
No, I meant this 👇

<img width="1185" alt="Screen Shot 2021-11-05 at 12 23 20 PM" src="https://user-images.githubusercontent.com/64056131/140573994-915708cf-acc6-4849-b9b5-02686c05e15e.png">

**Reply by Pheggas** [reply]
Daaaamn! Yep, that fullfills everything this feature request was about! Really looking forward to v1.0! Even now it starts to look better than 80% of open source software! 

Maybe one more question. As i asked on discord already. I saw contributors are welcomed and one of the types of contributions are popup texts for buttons. I can do this! Just need a little bump, like example how can i add a popup to specific button.
